### PR TITLE
Minor updates to form-related components.

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Issue numbers: #273 and #261 

## **This PR does the following:**
- Adds some changes so I could get `Label`, `Input`, `HelperErrorText`, and `Checkbox` to work in the Library Card app. Specifically:

* The `HelperErrorText` component wasn't exported.
* The Library Card app uses refs to focus on input elements for specific errors. So if there are 10 input elements on the page and the 6th one has an error because the email (for example) is invalid, the app creates a dynamic anchor error link to that input. But this doesn't seem to work if the `Input` component doesn't implement `forwardRef` to pass down the `ref` but not as a prop.
* Used the `attributes` prop in the `Input` component.
* Fixed the issue with the prop table not showing up in the "Info" tab in Storybook. The webpack config was removed...
* Updated the stories for some of the components.

Let me know if I'm missing anything else for this PR. I'll create a draft PR in the Library Card app that uses these changes.

### Front End Review:
- [ ] View [the example in Storybook]()

### CODE REVIEW

**Review by helenvholmes — APPROVED** [review]
This looks really good to me! Thanks for the updates and sticking to our AC-examples 😂

I did pop off some of the stuff from `Input` story #1 that was carried into #3, I think it's helpful to not have everything under the sun in the knobs panel if it's being illustrated in a different story.

I want @hokei to take a look at the `ref`s work you've done here too so I'm going to ask you don't merge just yet even though you have approval ✨

File: webpack.config.js
Comment (hokei) [inline webpack.config.js:41]: 😮

File: src/components/HelperErrorText/HelperErrorText.tsx
Comment (hokei) [inline src/components/HelperErrorText/HelperErrorText.tsx:26]: Do we want this to be knob-able?

Comment (helenvholmes) [reply src/components/HelperErrorText/HelperErrorText.tsx:26]: Yeah, I think we should... I can push up that change

[EDIT] We don't have knobs for HelperErrorText yet! Let's do that separately from this work.

File: src/components/Input/Input.tsx
Comment (hokei) [inline src/components/Input/Input.tsx:97]: With attributes added into inputProps, such that anyone can pass any extra prop and have it attached to `<input>`, could you technically go 

<DS.Input attributes={{ref: newRef}} /> and should do that instead, 

Or is it still something we want to do to pass refs into all the form components?  (I think ReactRouter allows explicit passing of refs, so I'm up for either).

File: src/components/HelperErrorText/HelperErrorText.tsx
Comment (EdwinGuzman) [reply src/components/HelperErrorText/HelperErrorText.tsx:26]: I had trouble setting up a knob for this because of the type.

File: src/components/Input/Input.tsx
Comment (EdwinGuzman) [reply src/components/Input/Input.tsx:97]: It makes sense and it is technically possible to do the above, but `ref` isn't an HTML input attribute and is mostly used for React. So I would like to keep that separate. I think it also makes sense to keep it separate since it's using forwardRef for this purpose.

**Comment by EdwinGuzman** [comment]
I pushed another commit that also adds the forwardRef implementation to the `Checkbox` component.

**Comment by EdwinGuzman** [comment]
The Library Card PR using this change is up: https://github.com/NYPL/nypl-library-card-app/pull/48

Whenever both can be reviewed, that'd be great.

File: package.json
Comment (hokei) [inline package.json:3]: this no longer should be bumped manually

**Pull request merged**
